scene/resources/animation.cpp

```diff
@@ -83,9 +83,11 @@ bool Animation::_set(const StringName &p_name, const Variant &p_value) {
 
 				const real_t *r = values.ptr();
 
-				tt->transforms.resize(vcount / 12);
+				//12 = TransformKey variable count (10) + Key variable count (2)
+				//4 = sizeof(real_t)
+				tt->transforms.resize(vcount / (12 * 4));
```

Comment:
What if `real_t` will be double not float?

Reply:
Check again. I did the same usage in the `_get` function